glean.1.schema.json

```diff
@@ -1,596 +1,600 @@
 {
-  "$id": "moz://mozilla.org/schemas/glean/ping/1",
```

Comment:
That's an awful lot of changes in here for what should be a small change.
Please update it upstream first: https://github.com/mozilla-services/mozilla-pipeline-schemas/
We have a script to update it from there: `bin/update-schema.sh`